daffodil-core/src/main/scala/edu/illinois/ncsa/daffodil/dpath/Expression.scala

```diff
@@ -294,8 +294,8 @@ case class ComparisonExpression(op: String, adds: List[Expression])
       case ("=", _) => subsetError("Unsupported operation '%s'. Use 'eq' instead.", op)
       case ("!=", _) => subsetError("Unsupported operation '%s'. Use 'ne' instead.", op)
 
-      case("eq", HexBinary) => EQ_CompareByteArray
-      case("ne", HexBinary) => NE_CompareByteArray
+      case ("eq", HexBinary) => EQ_CompareByteArray
+      case ("ne", HexBinary) => NE_CompareByteArray
```

Comment:
Testing commenting on a code hunk.

Reply:
Thank you for that useful comment. (Just a test of comments and replies while I figure out this code-review system.)

Reply:
Making a **bold** comment. :1st_place_medal: